ophyd/utils/epics_pvs.py

```diff
@@ -336,3 +341,16 @@ def wrapper(self, *args, **kwargs):
         else:
             raise DisconnectedError('{} is not connected'.format(self.name))
     return wrapper
+
+
+def set_and_wait(signal, val):
+    """
+    Set a signal to a value and wait until it reads correctly.
+
+    There are cases where this would not work well, so it should be revisited.
+    """
+    signal.put(val)
+    while signal.get() != val:
+        ttime.sleep(0.1)
+        logger.info("Waiting for %s to be set...", signal.name)
```

Comment:
A bit more info would be nice (current_value, set_value). Also, because I'm in that mood, there's a stray EOL at EOF